contracts/token/TransferManager.sol

```diff
@@ -183,7 +183,34 @@ contract TransferManager is Pausable {
 
         revert("Transfer not possible");
     }
-
+    
+    /**
+    * 
+    *  Require that the from and to addresses are not frozen.
+    *  Require that the value should not exceed available balance .
+    *  Require that the to address is a verified address,
+    *  If the `to` address is not currently a shareholder then it MUST become one.
+    *  If the transfer will reduce `from`'s balance to 0 then that address
+    *  MUST be removed from the list of shareholders.
+    *
+    * @param _from The address of the sender
+    * @param _to The address of the receiver
+    * @param _value The number of tokens to transfer
+    *
+    * @return `true` if successful and revert if unsuccessful
```

Comment:
I think you can remove the return boolean. If the function reverts it has failed, otherwise it was a success.

Reply:
well, that's true but the standard `transfer` and `transferFrom` functions in ERC20 are working with a bool return, so i liked the idea to work the same way with `forcedTransfer` even if it doesn't change much indeed

Reply:
Oh, you're right! Yep, we should follow the same standard!